docs/components/jupyter_dask_cloud_to_cluster.md

```diff
@@ -0,0 +1,7 @@
+#
+
+{% include-markdown "https://raw.githubusercontent.com/RS-DAT/JupyterDaskCloud2Cluster/main/README.md" start="# JupyterDaskCloud2Cluster" heading-offset=2 %}
+
+## Source code of the tool
+
+The source code of this tool can be found at [RS-DAT/JupyterDaskCloud2Cluster](https://github.com/RS-DAT/JupyterDaskCloud2Cluster).
```

Comment:
I love the idea of including the README from the various repositories and link to the GH repos for more information! I would however put the links to the repo at the top, so that this is immediately visible. Also maybe we can use the GH badge in the following way? 

```suggestion

GitHub repository: [![](https://img.shields.io/badge/github-repo-000.svg?logo=github&labelColor=gray&color=blue)][link]

{% include-markdown "https://raw.githubusercontent.com/RS-DAT/JupyterDaskCloud2Cluster/main/README.md" start="# JupyterDaskCloud2Cluster" heading-offset=2 %}

[link]: https://github.com/RS-DAT/JupyterDaskCloud2Cluster
```